app/components/businessType-list.component.css

```diff
@@ -111,6 +111,9 @@
 .bp-permit-name {
     color: gray;
 }
+a.bp-permit-name {
+  text-decoration: underline;
```

Comment:
@Bjorn248 I added a simple style to make it possible to see which permits have links. We might change later to be a cute little icon after the name of permit or similar, but at least an underline gets the job done.